tests/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27,py35,py36,py37
```

Comment:
@andrewschoen we have moved out of py27 and anyway its just couple of months for EOL for 27, Can we remove py27?

Reply:
Sure, I can do that. I ran this with ``py37`` locally anyway.